SA/bibliography.bib

```diff
@@ -1,3 +1,9 @@
+@online{enso-language,
+	author = {{Enso International Inc}},
```

Comment:
Are the nested `{` by intentions?

Reply:
The nested `{}` prevents the author name form being shortened in the bibliography

Reply:
hu, ok, I didn't know that 👍